troposphere/static/js/components/modals/instance/launch/components/Image.react.js

```diff
@@ -44,7 +44,7 @@ export default React.createClass({
         let activeTags = stores.TagStore.getImageTags(this.props.image);
 
         return (
-        <Tags tags={tags} activeTags={activeTags} renderLinks={false} />
+        <Tags activeTags={activeTags} renderLinks={false} />
```

Comment:
You can see below that the component used here doesn't reference the prop `tags`.
<img width="488" alt="viewtags" src="https://cloud.githubusercontent.com/assets/7366338/19085229/6fb8c9d2-8a1e-11e6-8813-e7a2e20153f4.png">